.github/workflows/catalogs.yml

```diff
@@ -168,6 +168,27 @@ jobs:
           path: nebius.csv
           retention-days: 1
 
+  catalog-cudocompute:
+    name: Collect CudoCompute catalog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Please, bump versions to

```
- uses: actions/checkout@v4
- uses: actions/setup-python@v5
```

Similar to other jobs in this file

Reply:
Done